bin/update_all_repos.sh

```diff
@@ -66,5 +66,8 @@ if [[ $failed = 0 ]]; then
   echo $(date) > /usr/local/repo/osg/timestamp.txt
 fi
 
-exit $failed
+# Update cadist under /usr/local/repo/cadist
+# Updates and errors go to /var/log/repo-update-cadist.{stdout,stderr}
+flock -n /var/lock/repo-update-cadist /usr/bin/repo-update-cadist
```

Comment:
@matyasselmeci @mwestphall what do you think about moving this up to line 64 and adjusting `$failed` based on the output as appropriate? I think the timestamp should reflect the last successful run of this whole process, which should probably include the cadist generation

Reply:
That makes sense, I'll update the script